pumpkin/src/world/mod.rs

```diff
@@ -53,6 +54,18 @@ type ChunkReceiver = (
     Receiver<Arc<RwLock<ChunkData>>>,
 );
 
+#[derive(Debug, Error)]
+pub enum WorldError {
```

Comment:
Also horrible name

Reply:
These errors were meant to be used for other Errors as well so we woudln't end up with infinitely many different error enums per file, but I guess they could be a bit more fine-grained